src/Symfony/Component/Form/Extension/Core/Type/CheckboxType.php

```diff
@@ -59,6 +59,7 @@ public function configureOptions(OptionsResolver $resolver)
             'value' => '1',
             'empty_data' => $emptyData,
             'compound' => false,
+            'false_values' => array(null),
```

Comment:
I think we should also force the option to be an array to not break the type hint of the transformer's constructor

Reply:
fixed